ipfx/dataset/create.py

```diff
@@ -73,12 +77,16 @@ def create_ephys_data_set(nwb_file: str,
 
     if not ontology_file:
         ontology_file = StimulusOntology.DEFAULT_STIMULUS_ONTOLOGY_FILE
+    ontology = StimulusOntology(ju.read(ontology_file))
 
     if nwb_version["major"] == 2:
+
         if is_mies:
-            nwb_data = MIESNWB2Data(nwb_file, ontology_file)
+            labnotebook = LabNotebookReaderIgorNwb(nwb_file)
+            nwb_data = MIESNWB2Data(nwb_file, labnotebook, ontology)
+
         else:
-            nwb_data = HBGNWB2Data(nwb_file, ontology_file)
+            nwb_data = HBGNWB2Data(nwb_file, ontology)
 
     else:
         raise ValueError("Unsupported or unknown NWB major" +
```

Comment:
use formatting here rather than +

Reply:
(I realize this is not your change, but in the interests of good stewardship would be nice, not a requirement to merge).